src/widgets/programs/programs-form.component.tsx

```diff
@@ -48,7 +49,7 @@ export default function ProgramsForm(props: ProgramsFormProps) {
   useEffect(() => {
     if (patientUuid && !viewEditForm) {
       const abortController = new AbortController();
-      getSession(abortController).then(({ data }) => {
+      getSession(abortController).then(({ data }: { data: SessionData }) => {
```

Comment:
We don't need the type here - the response is already fully typed in `getSession`.